scss/_mixins.scss

```diff
@@ -15,6 +15,36 @@
 	@return $aggregated-selector;
 }
 
+/// Sets SVG background retrieved from the image service
+@mixin oFtIconsGetSvg($icon-name, $color: null, $size: 20, $icon-only: false) {
+	@if ($icon-only == false) {
+		display: inline-block;
+		width: $size + 0px;
+		height: $size + 0px;
+	}
+
+	$service-url: 'https://next-geebee.ft.com/image/v1/images/raw/';
```

Comment:
it should, I'll raise an issue with missing implementation. The default will be o-assets if color is null, image service otherwise